### PR TITLE
Updated the readme to remove coding coach line

### PR DESCRIPTION
**Pull request by BrianMarist**
Un-needed information in the readme, was part of coding coach project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,6 @@ for marketing and is only seen by people you authorize.
 answers you seek
 - Listens: To its users, the sum of its parts, always remains a force for good.
 
-### You can learn more about the pedagogical approaches that agora is designed to promote and try it out for yourself on the original running Agora platform, the Coding Coach, here: https://codingcoach.net
```
Comment (briangormanly) [inline README.md:22]: Agreed this can go.

**Review by briangormanly — APPROVED** [review]
Approved.

File: README.md (hunk 1 above)
Comment (cmarcusdimarco) [reply README.md:22]: Looks good to me too.

**Review by cmarcusdimarco — APPROVED** [review]
Approved!

**Pull request merged**